src/main/java/core/basesyntax/serviceimpl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService<List<String>> {
+    private final List<String> lines = new ArrayList<>();
+
+    @Override
+    public List<String> readFromFile(String filePath) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
+            reader.readLine();
+            String line;
+            while ((line = reader.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Failed to read from file " + filePath, e);
+        }
+        return lines;
```

Comment:
The method readFromFile should not depend on a class field to store its state. This can lead to unexpected behavior when the method is called multiple times. Consider moving the 'lines' list inside the method to avoid shared state.